projects/js-packages/storybook/storybook/projects.js

```diff
@@ -0,0 +1,14 @@
+// List of projects paths that contains stories
+const path = require( 'path' );
+
+const join = relative => path.join( __dirname, relative );
+
+const projects = [
+	join( '../../base-styles' ),
+	join( '../../components/components' ),
+	join( '../../connection/components' ),
+	join( '../../idc/components' ),
+	join( '../../../packages/my-jetpack/_inc/components' ),
```

Comment:
since we are creating a new projects.js file, maybe we can reduce a simpler array? so we can define the project items without calling, one by one, the `join()`?

```es6
const projects = [
  '../../base-styles',
  '../../components/components',
  '../../connection/components',
  '../../idc/components',
  '../../../packages/my-jetpack/_inc/components',
];
```
take it as a simple suggestion

Reply:
@retrofox we can't do that because the storybook only resolves relative paths inside its structure since we're injecting paths outside of the structure, we need to use the absolute path.

Reply:
```es6
const projects = [
	'../../base-styles',
	'../../components/components',
	'../../connection/components',
	'../../idc/components',
	'../../../packages/my-jetpack/_inc/components',
];

module.exports = projects.map( path => join( path ) );
```

⬆️ is what I mean. It makes the projects array clearer.

Reply:
@retrofox oh! gotcha, looks cleaner, I will do that.

Reply:
done!